### ISSUE TITLE
Target is not instantiable while building

### ISSUE THREAD

**Issue by lofye** [stefanbauer/laravel-favicon-extractor#2]
I've added these to config/app.php
'providers' => [StefanBauer\LaravelFaviconExtractor\FaviconExtractorServiceProvider::class,...
'aliases' => ['FaviconExtractor' => StefanBauer\LaravelFaviconExtractor\Facades\FaviconExtractor::class...

And I'm getting the following error when trying to:
use StefanBauer\LaravelFaviconExtractor\Facades\FaviconExtractor;
$favicon = FaviconExtractor::fromUrl('https://laravel.com')->fetchOnly();

Illuminate\Contracts\Container\BindingResolutionException  : Target [StefanBauer\LaravelFaviconExtractor\Provider\ProviderInterface] is not instantiable while building [StefanBauer\LaravelFaviconExtractor\FaviconExtractor].

vendor/laravel/framework/src/Illuminate/Container/Container.php:958
    954|         } else {
    955|             $message = "Target [$concrete] is not instantiable.";
    956|         }
    957| 
  > 958|         throw new BindingResolutionException($message);
    959|     }

**Comment by alexbowers** [comment]
What version of Laravel are you using this with?

**Comment by neoteknic** [comment]
Same bug L 5.8 php 7.3

**Comment by jrebs** [comment]
I am having the same problem with laravel 5.8.32 and php 7.3.8. I did some experimentation and discovered a few things that might be of help diagnosing it (I have yet to figure out exactly why it is this way).

After a simple installation and then adding the facade to my `config/app.php`, I get the error that others have reported when I try utilize the facade:

```
Illuminate/Contracts/Container/BindingResolutionException with message 'Target [StefanBauer/LaravelFaviconExtractor/Provider/ProviderInterface] is not instantiable while building [StefanBauer/LaravelFaviconExtractor/FaviconExtractor]
```

If I look at the config data (`config('favicon-extractor')`), I see the expected value:

```
[
  "provider_class" => "StefanBauer\LaravelFaviconExtractor\Provider\GoogleProvider",
  "filename_generator_class" => "StefanBauer\LaravelFaviconExtractor\Generator\FilenameGenerator",
]
```

Where it gets strange is that I found that if you publish the configuration file to `config/favicon-extractor.php`, then suddenly the facade works.

I also found that if you do not publish the config but instead cache the application configurations with `php artisan config:cache` then it also causes the facade to work.

Hopefully this is of some help in diagnosing the issue.
